app/policies/material_shared_policy.rb

```diff
@@ -9,6 +9,11 @@ def edit_settings?
     admin? || user && (user.admin_for_projects & record.projects).length > 0
   end
 
+  def edit_credits?
+    # Admin or admin of any project.
+    admin? || project_admin?
```

Comment:
This should only allow the project admin of the current record. This line is also doing that: https://github.com/concord-consortium/rigse/blob/master/app/policies/material_shared_policy.rb#L9 So it seems like a method named `project_admin_of_material` should be created. So the logic can be shared.